### ISSUE TITLE
Adding RedwoodJS framework to the list supported frameworks

### ISSUE THREAD

**Issue by adriatic** [vercel/turbo#3425]
### Which project is this feature idea for?

Turborepo

### Describe the feature you'd like to request

I would like to be able to run the [create-turbo](https://turbo.build/repo/docs/getting-started/create-new#1-running-create-turbo) command and use a different **initial** framework (different than nextjs that is). 

Realizing that you already have support for several frameworks - like [svelte](https://github.com/vercel/turbo/tree/main/examples/with-svelte), these can **only be added** to the initial content of the monorepo (nextjs framework). I would like to get the initial monorepo host [RedwoodJS](https://redwoodjs.com/) and then continue adding various packages from the IoT context.

Is the approach documented as [Add Turborepo to your existing project](https://turbo.build/repo/docs/getting-started/add-to-project) equally valuable as my proposal "Adding RedwoodJS framework to the [list examples](https://github.com/vercel/turbo/tree/main/examples)? 

In summary:

1. Is adding RedwoodJS compatible with Vercel's future vision?
2. Would Vercel be interested in a contribution that would do the necessary work for item  1. above?

### Describe the solution you'd like

Too soon for that as my understanding of how to do this depends on the comments I hope to get from you.

### Describe alternatives you've considered

Too soon for that as my understanding of how to do this depends on the comments I hope to get from you.

**Comment by mehulkar** [comment]
@adriatic feel free to contribute to the `examples/` directory, a PR would be great! `create-turbo` doesn't currently support templates, it has only a single template with some variations for using `npm`/`yarn`/`pnpm`, so it probably can't support what you're asking for.

**Comment by anthonyshew** [comment]
As mentioned by Mehul, feel free to open a PR so users could use `npx create-turbo@latest -e with-redwood` and get started.

Short of that, though, I'll get this closed for inactivity!
